front/assets/js/project_onboarding/new/pages/workflow_setup/existing_configuration.tsx

```diff
@@ -75,7 +52,7 @@ export const ExistingConfiguration = () => {
                 <div className="mv3">
                   <a
                     href="#"
-                    onClick={(e) => { e.preventDefault(); void handleUseExisting(); }}
+                    onClick={(e) => { e.preventDefault(); handleUseExisting(); }}
```

Comment:
WDYT:
Rename the callback to `onHandleUseExisting`, `onClick={onHandleUseExisting}`, and then prevent the event default in the `onHandleUseExisting` callback.